### DISCUSSION TITLE
Elements that usually react to the beat stop reacting under certain conditions

### DISCUSSION THREAD

**Discussion by nagi-desuuu** [ppy/osu#20272] [Q&A]
Directly related (and possible continuation) to the issue: https://github.com/ppy/osu/issues/20059

After the newest Lazer update, which is 2022.911.0 at the time of writing, this issue seems to arise again, as demonstrated [here](https://streamable.com/mh2kqu).
Although rewinding the track or just letting it play eventually [fixes itself](https://streamable.com/zaziak).

(Side note: I would've classified this as an actual issue, but I forgot to take the logs with me when I turned off the computer.)

**Comment by Theighlin** [comment]
If you know an issue is happening again you can post it on the issue thread and if the bug is confirmed to still be occurring it will be reopened. (This discussion is here already so no need for that this time i think)
